target/linux/ath79/image/generic-ubnt.mk

```diff
@@ -243,6 +252,15 @@ define Device/ubnt_picostation-m
 endef
 TARGET_DEVICES += ubnt_picostation-m
 
+define Device/ubnt_powerbeam-5ac-500
+  $(Device/ubnt-xc)
+  SOC := qca9558
+  DEVICE_MODEL := PowerBeam 5AC
+	DEVICE_VARIANT := 500
+  DEVICE_PACKAGES := kmod-ath10k-ct-smallbuffers ath10k-firmware-qca988x-ct rssileds
```

Comment:
128 MB RAM should be enough for full kmod-ath10k-ct?
rssileds can be removed.

Reply:
Yes, it should be more than enough.